{
	"id": "CVE-2023-29418",
	"description": "An issue was discovered in libbzip3.a in bzip3 before 1.2.3. There is an xwrite out-of-bounds read.",
	"debian_bug": 1034177,
	"scope": "local",
	"packages": [
		{
			"name": "bzip3",
			"status": "resolved",
			"urgency": "not yet assigned",
			"fixed_version": "1.2.2-2",
			"repository": [
				{
					"name": "trixie",
					"version": "1.4.0-1"
				}
			]
		}
	]
}
